recaptcha_enterprise/pom.xml

```diff
@@ -5,7 +5,6 @@
   <artifactId>recaptcha-enterprise-snippets</artifactId>
   <packaging>jar</packaging>
   <name>Google reCAPTCHA Enterprise Snippets</name>
-  <url>https://github.com/googleapis/java-recaptchaenterprise</url>
```

Comment:
Should we link it to:
https://github.com/GoogleCloudPlatform/java-docs-samples/tree/main/recaptcha_enterprise